src/test/java/com/softserveinc/ita/HomePageTest.java

```diff
@@ -19,4 +21,20 @@ public void verifyLanguageSwitchingTest() {
                 .as("Language should be switched")
                 .isTrue();
     }
+
+    @Test
+    public void verifySearchTest() {
+        var requiredItem = "Xiaomi Redmi Note 10";
+        homePage.searchBarInputField(requiredItem);
```

Comment:
use chaining
homePage
.searchBarInputField(requiredItem)
.clickSearchButton();

Reply:
it doesn't work like that
![image](https://user-images.githubusercontent.com/83767551/142983881-2e94e08a-9ffb-4dbe-9821-1cdac41f5771.png)